src/screens/Dashboard/DashboardScreen.tsx

```diff
@@ -183,10 +187,11 @@ class DashboardScreen extends Component<Props, State> {
           onSendPress={this.sendCoins}
           onRecoveryPress={this.recoverCoins}
         />
-        {isAllWallets(activeWallet) ? (
+        {wallets.length > 0 ? (
```

Comment:
this condition is not need because this view won't be rendered without wallets
<img width="518" alt="Zrzut ekranu 2020-10-20 o 09 31 52" src="https://user-images.githubusercontent.com/67273035/96554645-1b397e80-12b7-11eb-91b8-ecf69a5a8106.png">

Reply:
hmm not sure can chaeck

Reply:
up

Reply:
done